src/main/java/frc/robot/superstructure/Superstructure.java

```diff
@@ -271,13 +293,14 @@ public Command transferElevatorShooter() {
 			flywheelStateHandler.setState(FlywheelState.DEFAULT),
 			elevatorStatesHandler.setState(ElevatorStates.IDLE),
 			pivotStateHandler.setState(PivotState.UP)
-		);
+		).handleInterrupt(() -> enableChangeStateAutomatically(true).schedule());
 	}
 
 	public Command intakeOuttake() {
 		AtomicReference<Double> lastTimeDetectedNote = new AtomicReference<>(TimeUtils.getCurrentTimeSeconds());
 		return new ParallelCommandGroup(
 			setCurrentStateValue(RobotState.INTAKE_OUTTAKE),
+			enableChangeStateAutomatically(true),
```

Comment:
I don't think it's necessary, but whatever you think
Like if you don't wanna use the motion planner here and you didn't do a handle interrupt I don't think you should put this here
Ur choice